frontend/src/components/housing/submission/SubmissionListNavigator.vue

```diff
@@ -255,13 +255,13 @@ function updateQueryParams() {
       </template>
     </Column>
     <Column
-      field="contactFirstName"
+      field="contacts.0.firstName"
       header="First name"
       :sortable="true"
       style="min-width: 150px"
     />
     <Column
-      field="contactLastName"
+      field="contacts.0.lastName"
```

Comment:
I guess we're assuming that the first contact will always be the primary?

Reply:
This is more a stylistic placeholder. I am building the back end to support multiple contacts even if the front end currently does not. I am assuming once we have to handle multiple contacts there will be a way to make a certain contact as the primary, at which point this will all have to be adjusted. Most likely partitioned into its own component and wrapped in a field array.